### ISSUE TITLE
Multiple accounts

### ISSUE THREAD

**Issue by retrov** [audiconnect/audi_connect_ha#186]
Been trying to add a second account to the integration but it seems to crash due to duplicate sensor names.
So when I go to the new added device it actually goes to the first car that was added.
Anyone seen this or have a way around it ?

Also tried to rename the entities before adding the second..

**Comment by retrov** [comment]
Yes.

**Comment by cdnninja** [comment]
To clarify you have two accounts with different cars?

**Comment by retrov** [comment]
Two accounts, one for each car. But since I'm a user of both I have access to both cars from one login. So it starts mixing up the entities from both cars. One entity might be from one car and another one from the other.

**Comment by retrov** [comment]
Tried it again today, and it's still picking up entities from each car and presenting them as one. Expected behaviour would be two devices with each of its entitites. It seems to have picked up all of the entities of my Etron 50, but the entities that this car doesn't have, ie Sunroof, is picked from my Etron 55.

**Comment by retrov** [comment]
I've attached the debug from the init of the plugin, I see that it get's the data from both cars.

[audiconnect_debug.txt](https://github.com/audiconnect/audi_connect_ha/files/14817533/audiconnect_debug.txt)

**Comment by retrov** [comment]
Anyone else seeing this with multiple cars on the account?

**Comment by retrov** [comment]
Yes, still a problem for me too.  It was working for a short time last year but haven't been able to replicate it with the older versions.
